README.md

```diff
@@ -8,7 +8,7 @@ Overview
 ---------
 An open, adaptable email security platform for writing and running custom detection and response rules to block phishing attacks, hunt for threats, and more.
 
-Sublime Platform is completely free and self-hosted, which means your emails are completely controlled by you and never seen by a third party. For more information on the different versions available, including Enterprise and managed Cloud offerings, see the [Versions page](https://sublimesecurity.com/versions).
+Sublime Platform is free and self-hosted, which means your emails are completely controlled by you and never seen by a third party.
```

Comment:
This might make it sound like we're an email service. Perhaps: "...which means no email data ever leaves your environment."

Reply:
(because the cloud email provider is a 3rd party)